docs/content/config/index.md

```diff
@@ -0,0 +1,83 @@
+| `HD_IMPRINT_URL`      | -       | `https://md.example.com/imprint`  | The URL that should be linked as the imprint in the footer.                                                                                                                |
+
+**ToDo:** Add screenshots to illustrate custom name and custom logo.
+
+## Database
+
+We officially support and test these databases:
+- SQLite (for development and smaller instances)
+- PostgreSQL
+- MariaDB
+
+| environment variable   | default | example             | description                                                                                   |
+|------------------------|---------|---------------------|-----------------------------------------------------------------------------------------------|
+| `HD_DATABASE_DIALECT`  | -       | `postgres`          | The database dialect you want to use. This can be `postgres`, `mysql`, `mariadb` or `sqlite`. |
+| `HD_DATABASE_HOST`     | -       | `db.example.com`    | The host, where the database runs. *Only if you're **not** using `sqlite`.*                   |
+| `HD_DATABASE_PORT`     | -       | `5432`              | The port, where the database runs. *Only if you're **not** using `sqlite`.*                   |
+| `HD_DATABASE_NAME`     | -       | `hedgedoc`          | The name of the database to use. *Only if you're **not** using `sqlite`.*                     |
+| `HD_DATABASE_USER`     | -       | `hedgedoc`          | The user that logs in the database. *Only if you're**not** using `sqlite`.*                   |
+| `HD_DATABASE_PASS`     | -       | `password`          | The password to log into the database. *Only if you're **not** using `sqlite`.*               |
+| `HD_DATABASE_STORAGE`  | -       | `./hedgedoc.sqlite` | The location of the database file. *Only if you're using `sqlite`.*                           |
+
+## External services
+
+| environment variable   | default | example                             | description                                                                                                                          |
+|------------------------|---------|-------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| `HD_PLANTUML_SERVER`   | -       | `https://www.plantuml.com/plantuml` | The PlantUML server that HedgeDoc uses to render PlantUML diagrams. If this is not configured, PlantUML diagrams won't be rendered.  |
+| `HD_IMAGE_PROXY`       | -       | `https://image-proxy.example.com`   | **ToDo:** Add description                                                                                                            |
+
+## Media
+
+There are couple of different backends that can be used to host your images for HedgeDoc.
+
+- [Azure](media/azure.md)
+- [local filesystem](media/filesystem.md)
+- [imgur](media/imgur.md)
```

Comment:
```suggestion
- [Imgur](media/imgur.md)
```

Reply:
But this is how the write their own name?